tests/basic/pure_test.go

```diff
@@ -4340,3 +4340,86 @@ var _ = Describe("{CreateCloneOfTheFADAVolume}", func() {
 		appsValidateAndDestroy(contexts)
 	})
 })
+
+var _ = Describe("{DeployFADAAndFBDAAppsAndStopPortworx}", func() {
+	/*
+		https://purestorage.atlassian.net/browse/PTX-37401
+		1.Deploy FADA and FACD apps and parallely and stop portworx for 10 mins
+		2.After 10 mins make it up and check if the pods are running
+		3.Destroy the apps
+	*/
+	JustBeforeEach(func() {
+		StartTorpedoTest("DeployFADAAndFBDAAppsAndStopPortworx",
```

Comment:
This test shouldn't be bound specifically to FADA and FBDA, nor should it have a set app list. Can we make this more like the existing tests, where it will keep the same node reboot logic but work with more apps?